db/schema.rb

```diff
@@ -11,7 +11,10 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160312201127) do
+ActiveRecord::Schema.define(version: 20160319021924) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
```

Comment:
Ugh. Its strange that this shows up now. I don't know enough about it but its worth keeping an eye on.